testdata/pki/cassandra.crt

```diff
@@ -1,83 +1,31 @@
-Certificate:
```

Comment:
I didn't realize until later that the original certs embedded text; I can add this if people think its useful, but one can always extract this using `openssl x509 -in testdata/pki/cassandra.crt -text -noout`

Reply:
:+1: agreed, was just a matter of including `-text` so added that!

Reply:
also a bit useful to see some subtle differences with the new cert, like the inclusion of SPIFFE based SANs.